package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
```

Comment:
I'd be fine with that, but it sounds like we'd need to rename when npm 4 releases.